src/Cassandra.IntegrationTests/project.json

```diff
@@ -45,7 +49,10 @@
         "System.Reflection": "4.1.0",
         "System.Reflection.Extensions": "4.0.1",
         "System.Reflection.Primitives": "4.0.1",
-        "Microsoft.DotNet.InternalAbstractions": "1.0.0"
+        "Microsoft.DotNet.InternalAbstractions": "1.0.0",
+        "System.Runtime.Serialization.Json": "4.0.2",
```

Comment:
These 3 dependencies: `System.Runtime.Serialization.Json`, `System.Runtime.Serialization.Primitives` and `System.Net.Http` can be moved above to the common `dependencies` property and removed from the framework-specific dependencies.